### DISCUSSION TITLE
Question about AdBloc

### DISCUSSION THREAD

**Discussion by bat999** [bromite/bromite#966] [Q&A] [answered]
Hi
Recently there have been two issues about AdBlock with Bromite browser.

This one...
"Ad block not working..."
https://github.com/bromite/bromite/issues/960
I'm OK with that one, it's fixed for me with version 88.0.4324.207.

But the other one...
"New Adblock list checker shows last checked time..."
https://github.com/bromite/bromite/issues/941
This one doesn't seem to be fixed (not for me anyway).
It shows Filter Version 2.2.0.76720
Last checked 29 Dec 1969 11:19.36

Please can somebody else confirm it's not fixed for them too.
Then maybe we can re-open issue 941.

I'm using Bromite version 88.0.4324.207.
My phone is Xiaomi Redmi Go.
The OS is ANDROID Oreo Go 8.1.0.

![Adblock_Screenshot_2021-03-02_11-48-08](https://user-images.githubusercontent.com/640761/109646573-4af36f80-7b50-11eb-886c-fbc54996eff5.jpg)

**Comment by csagan5** [comment, marked as answer]
@bat999 comment on that issue; the discussions are not for issues reporting or feature requests

**Reply by bat999** [reply]
OK
:-)

**Reply by bat999** [reply]
I've added a comment to issue #941
